content/en/security/application_security/enabling/single_step/code_security.md

```diff
@@ -9,7 +9,7 @@ kind: documentation
 ## Requirements
 
 - **Minimum Agent version 7.53.0**
-- **Languages and architectures**: Single step ASM instrumentation only supports tracing Java, Python, Ruby, Node.js, and .NET Core services on `x86_64` and `arm64` architectures.
+- **Languages and architectures**: Single step ASM instrumentation for code security only supports tracing Java, Node.js, python (private beta), and .NET Core services on `x86_64` and `arm64` architectures.
```

Comment:
Is this intentionally lower case compared to other sections with uppercase Python?

```suggestion
- **Languages and architectures**: Single step ASM instrumentation for code security only supports tracing Java, Node.js, Python (private beta), and .NET Core services on `x86_64` and `arm64` architectures.
```

Reply:
ah good catch, it should be upper case, I'll fix :)